File: algoritmos/03 - Estrutura condicional/Questões/questao06.py
# Questao 06

a = float(input("Insira o valor de A: "))
b = float(input("Insira o valor de B: "))
c = float(input("Insira o valor de C: "))

delta = (b**2) - (4*a*c)

if delta > 0:
    print("2 raízes")
    print(delta)
elif delta == 0:
    print("1 raíz")
    print(delta)
else:
    print("0 raízes")
    print(delta)

File: estrutura de dados/4 - Ponteiros/questoes/questao06.c
#include <stdio.h>

int main() {
    int a = 10, b = 0;
    int *pA = &a;

    b = *pA;

    printf("Valor de a: %d\n", a);
    printf("Valor de b: %d\n", b);

    return 0;
}
